Simplified/NYPLBookRegistry.m

```diff
@@ -383,7 +383,7 @@ - (void)syncWithStandardAlertsOnCompletion
       [self save];
     } else {
       UIAlertController *alert = [NYPLAlertUtils alertWithTitle:@"SyncFailed"
-                                                        message:@"CheckConnection"];
+                                                        message:@"LibraryLoadError"];
```

Comment:
Since this is getting a loans feed, not a list of libraries, the error message to use is DataError, not LibraryLoadError.